src/ui/Tab/Tab.css.ts

```diff
@@ -0,0 +1,49 @@
+import { style } from "@vanilla-extract/css";
+
+import { sprinkles } from "../sprinkles.css";
+import { theme } from "../tokens";
+
+export const tabRootCss = sprinkles({
+  alignItems: "center",
+  display: "flex",
+  flexDirection: "column",
+  gap: "lg",
+});
+
+export const tabListCss = style([
+  sprinkles({
+    display: "flex",
+    gap: "xs",
+    overflow: "scroll",
+    width: "full",
+  }),
+  {
+    "::-webkit-scrollbar": {
+      display: "none",
+    },
+  },
+]);
+
+export const typographyCss = style([
+  sprinkles({
+    backgroundColor: "white",
+    borderColor: "black7",
+    borderStyle: "solid",
+    borderWidth: "xxxxs",
+    color: "black3",
+    cursor: "pointer",
+    minWidth: "fit",
+    outline: "none",
+    px: "sm",
+    py: "xs",
+  }),
+  {
+    borderRadius: "5rem",
+    selectors: {
+      '&[data-state="active"]': {
+        border: `solid 1px ${theme.colors.orange2}`,
```

Comment:
```suggestion
        borderColor: theme.colors.orange2,
```
`active`일때 `borderColor`값만 변경되는 거라면 이렇게 작성해도 될것 같아요!

Reply:
수정완료 했습니다 :)